lib/hermit-git

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+#
+# Summary: Run git operations on current profile repository
+# Usage: hermit git <git arguments>
+
+# This is a pass-through from hermit to git. It doesn't implement
+# any additional functionality, just changes to the profile directory
+# and runs git with the arguments specified.
```

Comment:
If this section is intended to be the help documentation for the command then line 5 needs to have a leading 
"#".  The code that extracts help docs from file headers depends on all the help lines starting with a "#".